Automattic/jetpack
# E2E tests: Add mailchimp block test

### Changes proposed in this Pull Request:

* Add Mailchimp test which covers the m/c connection flow

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* p9dueE-Ju-p2

#### Testing instructions:






* Check Travis build. Make sure tests are passed
* Maybe run the test locally 🤷‍♂ 

#### Proposed changelog entry for your changes:

* n/a
